DiceKit/FrequencyDistribution.swift

```diff
@@ -216,4 +216,12 @@ extension FrequencyDistribution {
         }
     }
     
+    public func minOutcome() -> Int {
```

Comment:
I would just do `minimum` and then return the whole tuple (pseudo-code):

```swift
let minimumOutcome = orderedOutcomes.first
let minimumFrequency = frequenciesPerOutcomes[minimumOutcome]
return (minimumOutcome, minimumFrequency)
```

Or, since we have `index` now:

```swift
var minimumOutcomeIndex: Index {
...
}
```

Not sure really.